.travis.yml

```diff
@@ -6,5 +6,5 @@ cache:
   cocoapods: true
 
 script:
-  - set -o pipefail && xcodebuild -workspace BraintreeDropIn.xcworkspace -scheme UnitTests -destination platform\=iOS\ Simulator,OS\=13.3,name\=iPhone\ 11 test | bundle exec xcpretty -c
+  - set -o pipefail && xcodebuild -workspace BraintreeDropIn.xcworkspace -scheme DropInDemo -destination platform\=iOS\ Simulator,OS\=13.3,name\=iPhone\ 11 build test | bundle exec xcpretty -c
```

Comment:
The additional `build` command is needed now to build the demo app?

Reply:
It's not strictly necessary for running the tests. It's just checking that we can actually build the demo app.